### PR TITLE
CONCF-1047 | Add re-initialization on resize to cropper tool

### PR DESCRIPTION
**Pull request by vforge**
/cc @hakubo @rogatty 

see https://github.com/fengyuanchen/cropper/issues/421#issuecomment-131293395
and https://github.com/fengyuanchen/cropper/pull/429

### CODE REVIEW

File: front/scripts/main/components/CuratedContentEditorImageCropComponent.ts [hunk 1]
```diff
@@ -104,11 +107,29 @@ App.CuratedContentEditorImageCropComponent = Em.Component.extend(
...
 				});
 
-			$imgElement.cropper(settings);
+			if (!this.get('cropperInitialized')) {
+				$imgElement.cropper(settings);
+
+				this.setProperties({
+					isLoading: false,
+					cropperInitialized: true,
+					$imgElement
```
Comment (rogatty) [inline front/scripts/main/components/CuratedContentEditorImageCropComponent.ts]: Please reuse this property instead of querying the DOM again.

Comment (vforge) [inline front/scripts/main/components/CuratedContentEditorImageCropComponent.ts]: Done, see 9a5a992

**Comment by hakubo** [comment]
LGTM

File: front/scripts/main/components/CuratedContentEditorImageCropComponent.ts [hunk 2]
```diff
@@ -99,16 +107,28 @@ App.CuratedContentEditorImageCropComponent = Em.Component.extend(
...
+			}
+		},
 
-			$imgElement.cropper(settings);
+		onResize(): void {
+			var $imgElement = this.get('$imgElement');
+
+			if (this.get('cropperInitialized')) {
+				// re-init cropper according to https://github.com/fengyuanchen/cropper/issues/421
+				$imgElement.cropper('destroy');
```
Comment (vforge) [inline front/scripts/main/components/CuratedContentEditorImageCropComponent.ts]: See also https://github.com/fengyuanchen/cropper/pull/429

**Pull request merged**